<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jedna macierz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: Arial;
	font-size: 24px;
}
#centralwidget{
	background-color: #0ca678;
}
QPushButton{
	height: 60px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
	background-color: transparent;
}
QPushButton:hover{
background-color: white; 
color: #0ca678;
}
QLabel{
	padding-right: 10px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
}
QLineEdit, QLineEdit:active{
	color: white;
	border: 1px solid white;
	background: #0ca678;
}
QComboBox{
	background: #0ca678;
	border: 2px solid white;
	color: white;
}
#matrix1widget, #matrix2widget, #matrix3widget{
border: 2px solid white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 0;</string>
    </property>
    <property name="text">
     <string>Wróć</string>
    </property>
   </widget>
   <widget class="QWidget" name="matrix1widget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboMatrixDims">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2x2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3x3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4x4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5x5</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>701</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridButtons">
     <item row="6" column="1">
      <widget class="QPushButton" name="btnComplement">
       <property name="text">
        <string>Macierz dołączona</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnRank">
       <property name="text">
        <string>Rząd macierzy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnTrace">
       <property name="text">
        <string>Ślad macierzy</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="btnTranspose">
       <property name="styleSheet">
        <string notr="true">font-size: 22px</string>
       </property>
       <property name="text">
        <string>Macierz transponowana</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="btnInverse">
       <property name="text">
        <string>Macierz odwrotna</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnDet">
       <property name="text">
        <string>Wyznacznik</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnNorm">
       <property name="text">
        <string>Normy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnCondition">
       <property name="text">
        <string>Kappa</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnSum">
       <property name="text">
        <string>Suma elementów</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>511</width>
      <height>249</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
